numpy/polynomial/chebyshev.pyi

```diff
@@ -87,40 +95,39 @@ def _zseries_div(
 def _zseries_der(zs: npt.NDArray[_NumberOrObjectT]) -> _Series[_NumberOrObjectT]: ...
 def _zseries_int(zs: npt.NDArray[_NumberOrObjectT]) -> _Series[_NumberOrObjectT]: ...
 
-poly2cheb: _FuncPoly2Ortho[L["poly2cheb"]]
-cheb2poly: _FuncUnOp[L["cheb2poly"]]
+poly2cheb: Final[_FuncPoly2Ortho] = ...
```

Comment:
The `= ...` indicate the presence of a value at runtime, and the `Final[_]` markers prohibit reassignment.